packages/docusaurus-theme-mermaid/package.json

```diff
@@ -39,6 +39,7 @@
     "@docusaurus/types": "3.8.1",
     "@docusaurus/utils-validation": "3.8.1",
     "mermaid": ">=11.6.0",
+    "@mermaid-js/layout-elk": "^0.1.8",
```

Comment:
Can you make this dependency optional?

This package is large, and there are also license concerns: https://github.com/mermaid-js/mermaid/pull/5654

We don't want existing Docusaurus sites under MIT license to implicitly change their license while upgrading to the next minor version